test/unit/virtual_include/test_virtual_include.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 Ericsson AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Unit test for resolving path in all plist files
+"""
+import logging
+import os
+import re
+import unittest
+import glob
+from typing import final
+from common.base import TestBase
+
+
+class TestVirtualInclude(TestBase):
+    """Tests checking virtual include path resolution"""
+
+    # Set working directory
+    __test_path__ = os.path.dirname(os.path.abspath(__file__))
+    BAZEL_BIN_DIR = os.path.join(
+        "../../..", "bazel-bin", "test", "unit", "virtual_include"
+    )
+    BAZEL_TESTLOGS_DIR = os.path.join(
+        "../../..", "bazel-testlogs", "test", "unit", "virtual_include"
+    )
+
+    @final
+    @classmethod
+    def setUpClass(cls):
+        """TODO: Define set up before the test suite"""
+        super().setUpClass()
+        cls.run_command("bazel clean")
+
+    @final
+    @classmethod
+    def tearDownClass(cls):
+        """TODO: Define clean up after the test suite"""
+        super().tearDownClass()
+
+    def setUp(self):
+        """TODO: Define clean up before every test"""
+        super().setUp()
+
+    def tearDown(self):
+        """TODO: Define clean up after every test"""
+        return super().tearDown()
```

Comment:
I don't think we need any of this. I'm still curious about `bazel clean` -- why do we need it? Can you explain a bit?

Reply:
Can we understand a bit better how that can happen?